src/orb.yml

```diff
@@ -36,7 +36,7 @@ commands:
       - run:
           name: Send Comment
           command: |
-            GH_LOGIN=$(curl -sS https://api.github.com/user\?access_token\=$GHI_TOKEN | jq '.login' --raw-output)
+            GH_LOGIN=$(curl -sS -H 'Authorization: token $GHI_TOKEN' https://api.github.com/user/repos | jq '.login' --raw-output)
```

Comment:
Hey, it seems to me that the `/repos` addition causes this command to incorrectly pick up `null` as the user login since `/repos` gives an array. The log is `Authenticated with null` and then the bot re-comments each time because it can't find any previous comment. Has this new command been working for you?